apps/modernization-ui/src/apps/search/advancedSearch/AdvancedSearch.scss

```diff
@@ -1,5 +1,15 @@
 @use 'styles/colors';
 
+.advanced-search-title {
+    font-size: 100%;
```

Comment:
issue:  this font-size is causing the "Search" `h1` to render smaller than it should be.

Reply:
![Screenshot 2024-01-30 at 6 29 14 PM](https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/1dce5148-7d13-4698-ba0d-29ad5c99f1b5)

This is the current design on demo so I reversed engineered it to be the same size. We need clarification at the new headers can break the current font sizes in the design.